proposals/lip-0043.md

```diff
@@ -112,8 +112,9 @@ The `ownName` property sets the name of the sidechain in its own state according
 | Name                                  | Type   | Value                | Description                                        |
```

Comment:
Agree. In fact, since we switch to strings for most idetifiers, claiming that memorizing a number is an advantage does not make sense. I would just suggest removing the last paragraph of the Chain ID section.

Reply:
I updated the section